web/js/projection/projection.change.js

```diff
@@ -1,21 +1,16 @@
-/**
- * @module wv.proj
- */
-var wv = wv || {};
-wv.proj = wv.proj || {};
+import $ from 'jquery';
+import util from '../util/util';
+import wvui from '../ui/ui';
 
-/**
- * @class wv.proj.change
- */
-wv.proj.change = wv.proj.change || function (models) {
+export function projectionChange(models) {
```

Comment:
What do you think about keeping the name `change`, importing this into the `proj` parent, and attaching it to the object that exports?